app/src/main/java/com/nervousfish/nervousfish/ConstantKeywords.java

```diff
@@ -12,10 +13,10 @@
     public static final String PROXY_REQUIRED = "Proxy required.";
 
     /**
-     * Prevent instantiation
+     * Prevents instantiation from outside the class
      */
     private ConstantKeywords() {
-
+        // Unused, just prevents instantiation from outside the class
```

Comment:
Kind of redundant when the JavaDoc already specifies that and we're OK with h empty constructors